src/extension.ts

```diff
@@ -213,13 +227,21 @@ async function startStatusUpdateTimer(): Promise<void> {
   }, 1000);
 }
 
-function readPreset(crcStatus: Status): 'Podman' | 'OpenShift' | 'unknown' {
+function stopUpdateTimer(): void {
+  if (statusFetchTimer) {
+    clearInterval(statusFetchTimer);
+  }
+}
+
+function readPreset(crcStatus: Status): 'Podman' | 'OpenShift' | 'MicroShift' | 'unknown' {
   try {
     switch (crcStatus.Preset) {
       case 'podman':
         return 'Podman';
       case 'openshift':
         return 'OpenShift';
+      case 'microshift':
+        return 'MicroShift';
```

Comment:
Perhaps later, when the name is 'clearer', we have to extract this into a mapping to allow these names to be changed from a single file (or perhaps even provided from an API). Not to be changed now.